### ISSUE TITLE
Classification accuracy calculation Error in finetuning when multilabel situation

### ISSUE THREAD

**Issue by Norman-GM** [935963004/LaBraM#15]
Hi @935963004 , thanks for opening the excellent work! 

As title illustrated, in finetuning phase line 117, the training accuracy is calculated as the class_acc = (output.max(-1)[-1] == targets).float().mean() , since the output has the tensor with dimension (64, 3) and target (64, 1), after max(-1)[-1], the pred dimension is (64), directly calling == will use the pytorch broadcasting mechanism, 
- pred broadcast expanded from [64] to [1, 64]
- target expanded from [64, 1] broadcast to [64, 64]
As a result, both tensors are expanded to [64, 64], and then the two expanded tensors are compared element-wise. Therefore, each pred[i] element is compared with an element of each row of target[:, 0] in target, and the end result is a [64, 64] tensor representing the comparison result for each combination. 
**The correct approach is as follows**：
 simply replace with the (output.max(-1)[-1] == targets.squeeze()).float().mean()
OR
just as the val and test processing:
 class_acc = utils.get_metrics(output.detach().cpu().numpy(), targets.detach().cpu().numpy(), ['accuracy'], is_binary)['accuracy']

Here is the **minimal reproducible code**: 

import torch
pred = torch.tensor([0, 0, 2, 1, 0, 2, 2, 2, 1, 1, 0, 2, 0, 2, 0, 0, 1, 1, 2, 0, 2, 0, 0, 2,
        0, 0, 0, 0, 0, 2, 2, 0, 1, 2, 0, 2, 0, 2, 1, 2, 0, 0, 1, 0, 2, 1, 2, 0,
        2, 1, 1, 2, 1, 1, 2, 2, 0, 1, 0, 0, 1, 1, 0, 2])
target = torch.tensor([[0],[0],[2],[1],[0],[2],[2], [2],[1],[1],[0],[2],[0],[2],[0],[1],[1], [1], [2], [0], [2], [1], [0], [2],[0],[0], [0],[0], [0], [2],[2],[0],[1],[2],
[0],[2],[0],[2],[1],[2],[0],[0],[1],[0],[2],[1], [2],[0],[2],[1],[1],[2], [1], [1],[2],[2],[0],[1],[0],[0],[1],[1],[0],[2]])

comparison = (pred == target).float()
comparison_squeeze = (pred == target.squeeze()).float()

accuracy = comparison.mean()
accuracy_squeeze = comparison_squeeze.mean()

accuracy.item(), accuracy_squeeze.item()

(0.3408203125, 0.96875)

**Comment by 935963004** [comment]
Thanks for fixing accuracy calculation bug! I have updated the code.
